<?xml version="1.0"?>
<launch>
    <arg name="nr" default="1"/> 
    <arg name="node_name" default="$(eval 'robot_' + str(arg('nr')))"/>
    <arg name="robot_id" default="1"/>
    <arg name="robot_capacity" default="5"/>

    <include file="$(find mobile_robot)/launch/robot.launch">
        <arg name="node_name" value="$(arg node_name)" />
        <arg name="robot_id" value="$(arg nr)" />
        <arg name="robot_capacity" value="$(arg robot_capacity)" />
    </include>

    <include file="$(find mobile_robot)/launch/multiple_robots.launch" if="$(eval arg('nr') - 1 > 0)">
        <arg name="nr" value="$(eval arg('nr') - 1)"/>
        <arg name="robot_capacity" value="$(arg robot_capacity)" />
    </include>
</launch>

